components/org.wso2.carbon.identity.organization.management.service/src/main/java/org/wso2/carbon/identity/organization/management/service/dao/OrganizationManagementDAO.java

```diff
@@ -378,4 +378,13 @@ int getRelativeDepthBetweenOrganizationsInSameBranch(String firstOrgId, String s
      */
     String getAnAncestorOrganizationIdInGivenDepth(String organizationId, int depth)
             throws OrganizationManagementServerException;
+
+    /**
+     * Add a root organization.
+     *
+     * @param rootOrganization The root organization to be added.
+     * @throws OrganizationManagementServerException The server exception thrown when adding a root organization.
+     */
+    void addRootOrganization(Organization rootOrganization) throws OrganizationManagementServerException;
```

Comment:
make this default interface method

Reply:
Fixed by - https://github.com/wso2/identity-organization-management-core/pull/89/commits/9c30415a2ea1cfc93bd23a06f09254730216aee9